examples/s2n-mtls/src/bin/quic_echo_client.rs

```diff
@@ -0,0 +1,55 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+use std::{error::Error, net::SocketAddr, path::Path};
+
+use s2n_quic::{client::Connect, provider::tls, Client};
+
+pub static CA_CERT_PEM: &str = concat!(
+    env!("CARGO_MANIFEST_DIR"),
+    "/../rustls-mtls/certs/ca-cert.pem"
+);
+pub static CLIENT_CERT_PEM: &str = concat!(
+    env!("CARGO_MANIFEST_DIR"),
+    "/../rustls-mtls/certs/client-cert.pem"
+);
+pub static CLIENT_KEY_PEM: &str = concat!(
+    env!("CARGO_MANIFEST_DIR"),
+    "/../rustls-mtls/certs/client-key.pem"
+);
```

Comment:
Can you just copy the certs into this example directory? I'd prefer each example be standalone :)